src/views/Dashboard/Lab/index.vue

```diff
@@ -92,15 +92,23 @@ import LabOrders from "./LabOrders";
 
 export default {
   name: 'Lab',
+
   components: {
     MenuCard,
     LabOrders,
   },
+
   computed: {
     ...mapState({
       labAccount: (state) => state.substrate.labAccount,
       isServicesExist: (state) => state.substrate.isServicesExist,
     }),
-  },
+
+    computeVerificationStatus() {
+      return this.labAccount?.verificationStatus
+        ? `Your account has been ${this.labAccount?.verificationStatus}`
```

Comment:
Try this instead `Your lab account's verification status is: ${this.labAccount?.verificationStatus}`

Reply:
thanks, sure will update asap